### PR TITLE
Change event signin page to fit in phone

### PR DESCRIPTION
**Pull request by audreygao**
How it looks on browser:
Screen Shot 2021-09-18 at 5 23 38 PM
Screen Shot 2021-09-18 at 5 24 07 PM
On Phone:
IMG_5292
Screen Shot 2021-09-18 at 5 24 54 PM

### CODE REVIEW

File: frontend/src/pages/EventSignInPage/styles.ts [hunk 1]
```diff
@@ -12,6 +12,16 @@ const useStyles = makeStyles({
     alignItems: 'center',
     width: '600px',
     paddingBottom: '30px',
+    [theme.breakpoints.up('sm')]: {
+      width: 600,
+      marginLeft: 'auto',
+      marginRight: 'auto',
+    },
+    [theme.breakpoints.down('sm')]: {
```
Comment (thai-truong) [inline frontend/src/pages/EventSignInPage/styles.ts:20]: Just out of curiosity, what does the up() and down() thing do? And what is the `breakpoints` thing? I've always been bad with CSS so just wanted to know : )

File: frontend/src/pages/EventSignInOptionsPage/index.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ function EventSignInOptionsPage(): JSX.Element {
   const guestSignInURL = ROUTES.EVENT_SIGN_IN_WITH_ID(eventId);
 
   return (
-    <div style={{ margin: '200px' }}>
```
Comment (thai-truong) [inline frontend/src/pages/EventSignInOptionsPage/index.tsx:24]: Nice catch!

**Review by thai-truong — APPROVED** [review]
Lgtm! Thanks for doing this, it looks so much nicer. I'm not very good at CSS so I say you should wait to the other reviewer to give an approval before merging, but definitely an approval from me.

File: frontend/src/pages/EventSignInPage/styles.ts (hunk 1 above)
Comment (audreygao) [reply frontend/src/pages/EventSignInPage/styles.ts:20]: it's creating different size media query based on the screen size, up would be targeting screen size larger than the given parameter, and down would be for screen size smaller than the parameter

**Comment by thai-truong** [comment]
@yuw006 can you review this PR since it should be quick?

**Comment by yuw006** [comment]
Just curious what happens when the details is really long?

**Comment by audreygao** [comment]
> Just curious what happens when the details is really long? @audreygao

like which part? It would just wrap the text I believe.

**Comment by yuw006** [comment]
> > Just curious what happens when the details is really long? @audreygao
> 
> like which part? It would just wrap the text I believe.

sounds good

**Review by yuw006 — APPROVED** [review]

**Comment by thai-truong** [comment]
You can merge this in now @audreygao

**Pull request merged**